conf/META-INF/persistence.xml

```diff
@@ -33,9 +33,9 @@
             <property name="eclipselink.logging.level" value="ALL" />
 
             <property name="javax.persistence.schema-generation.database.action" value="drop-and-create"/>
-            <property name="javax.persistence.schema-generation.scripts.action" value="drop-and-create"/>
-            <property name="javax.persistence.schema-generation.scripts.create-target" value="target/sampleCreate.ddl"/>
-            <property name="javax.persistence.schema-generation.scripts.drop-target" value="target/sampleDrop.ddl"/>
+            <!--<property name="javax.persistence.schema-generation.scripts.action" value="drop-and-create"/>-->
+            <!--<property name="javax.persistence.schema-generation.scripts.create-target" value="target/sampleCreate.ddl"/>-->
+            <!--<property name="javax.persistence.schema-generation.scripts.drop-target" value="target/sampleDrop.ddl"/>-->
```

Comment:
What's the impact of this change? Will the loader still create the schema if it doesn't yet exist? I don't think we yet have a way of creating the database schema other than this.